draco/analysis/flagging.py

```diff
@@ -2981,3 +2981,259 @@ def destripe(x, w, axis=1):
     bsel = tuple(bsel)
 
     return x - stripe[bsel]
+
+
+class SiderealMaskConversion(task.SingleTask):
+    """Convert the axis of an RFI mask from time to ra.
+
```

Comment:
It would be helpful to have a quick explanation here about the method this task uses to convert from time to RA.